src/repositories/likes.repository.ts

```diff
@@ -0,0 +1,32 @@
+import { LikeModel } from "../models/Like.model";
+
+export const getOne = async (
+  data: { _id: string } | { userId: string; tweetId: string }
+) => {
+  return LikeModel.findOne(data);
+};
+export const deleteOne = async (
```

Comment:
```suggestion
export const deleteOne =  (
```
no await in that function